lib/components/Utilities/IndicatorsList/index.tsx

```diff
@@ -0,0 +1,26 @@
+import { Indicator } from "@/ui/indicator"
+import { forwardRef } from "react"
+import { IconType } from "../Icon"
+
+export interface IndicatorsListProps {
+  items: {
+    label: string
+    count: number
+    icon?: IconType
+    color: string
+  }[]
```

Comment:
We could better get these attributes from the `Indicator` component props:

```suggestion
  items: IndicatorProps[]
```